SQL/MySQL.sublime-syntax

```diff
@@ -95,6 +95,11 @@ contexts:
     - meta_prepend: true
     - include: rename-statements
     - include: show-statements
+    - match: \b(?i:table)\b
+      scope: keyword.context.mysql
+      push:
+        - table-name
+        - single-identifier
```

Comment:
To follow other "identifier" use cases, which all ensure scope (e.g. `meta.table-name`) to start with first non-whitespace character, I'd suggest the following change:

```suggestion
      push: expect-table-name
```

Reply:
nice catch - done :)